install_config/aggregate_logging.adoc

```diff
@@ -2054,6 +2054,186 @@ $ oc exec <$espod> -- es_load_kibana_ui_objects <user-name>
+}
+----
+<1> Specify the `.operations.*` or a `project.` index pattern.
+
+. Get the name of one of the Elasticsearch pods:
++
+----
+$ oc get -n logging pods -l component=es
+
+NAME                                       READY     STATUS    RESTARTS   AGE       IP             NODE                         NOMINATED NODE
+logging-es-data-master-5av030lk-1-2x494    2/2       Running   0          38m       154.128.0.80   ip-153-12-8-6.wef.internal   <none>
+----
+
+. Load the JSON file into the Elasticsearch pod:
++
+----
+$ cat <json-file-name> | \
+oc exec -n logging -i -c elasticsearch <es-pod-name> -- \
+    curl -s -k --cert /etc/elasticsearch/secret/admin-cert \
+    --key /etc/elasticsearch/secret/admin-key \
+    https://localhost:9200/_template/<json-file-name> -XPUT -d@- | \
+python -mjson.tool
+----
+
+. Get the existing index pattern file from your Elasticsearch container:
++
+----
+$ mkdir index_patterns
+$ cd index_patterns
+$ oc project openshift-logging
+$ for espod in $( oc get pods -l component=es -o jsonpath='{.items[*].metadata.name}' ) ; do
+  for ff in $( oc exec -c elasticsearch <espod-name> -- ls /usr/share/elasticsearch/index_patterns ) ; do
+    oc exec -c elasticsearch <es-pod-name> -- cat /usr/share/elasticsearch/index_patterns/$ff > $ff
+  done
+  break
+done
+----
++
+The index patterns download to the *_/usr/share/elasticsearch/index_patterns_*
+directory.
+
+.  If you have a separate OPS cluster, get the name of one of the es-ops 
+Elasticsearch pods:
++
+----
+$ oc get -n logging pods -l component=es-ops
+
+NAME                                           READY     STATUS    RESTARTS   AGE       IP             NODE                         NOMINATED NODE
+logging-es-ops-data-master-o7nhcbo4-5-b7stm    2/2       Running   0          38m       154.128.0.80   ip-153-12-8-6.wef.internal   <none>
+----
+
+. Load the JSON file into the es-ops Elasticsearch pod:
++
+----
+$ cat <json-file-name> | \
+oc exec -n logging -i -c elasticsearch <esops-pod-name> -- \
+    curl -s -k --cert /etc/elasticsearch/secret/admin-cert \
+    --key /etc/elasticsearch/secret/admin-key \
+    https://localhost:9200/_template/<json-file-name> -XPUT -d@- | \
+python -mjson.tool
+----
++
+[NOTE] 
+====
+The settings do not apply to existing indices. If you want to apply the settings
+to existing indices, perform a reindexing.
+====
+
+. Get the existing index pattern file from your Elasticsearch container:
++
+----
+$ mkdir index_patterns
+$ cd index_patterns
+$ oc project openshift-logging
+$ for espod in $( oc get pods -l component=es -o jsonpath='{.items[*].metadata.name}' ) ; do
+  for ff in $( oc exec -c elasticsearch <esops-pod-name> -- ls /usr/share/elasticsearch/index_patterns ) ; do
+    oc exec -c elasticsearch <esops-pod-name> -- cat /usr/share/elasticsearch/index_patterns/$ff > $ff
+  done
+  break
+done
+----
++
+The index patterns are download to the *_/usr/share/elasticsearch/index_patterns_* directory.
+
```

Comment:
Would the user need to do this for the OPS cluster or would the index patterns be the same?